server/models/gamelogs.js

```diff
@@ -78,8 +79,85 @@ function addGame(date, playerOne, playerTwo, throwOne, throwTwo, winner) {
     })
 }
 
+function fetchPlayerWins(id) {
+  return new Promise((resolve, reject) => {
+    GameLogs.find({winner: id}, function (error, gamelog) {
+      if (error) {
+        reject(error)
+      }
+      resolve(gamelog)
+    })
+  })
+}
+
+function fetchPlayerLosses(id) {
+  return new Promise((resolve, reject) => {
+    GameLogs.find({
+      $or:[
+        {playerOne:id},
+        {playerTwo:id}
+      ]})
+      .where({ winner: { $ne: id } } )
+      .exec((error, gameLogs) => {
+        if (error) { reject(error) }
+        resolve(gameLogs)
+      })
+  })
+}
+
+function fetchPlayerThrows(id) {
+  return new Promise((resolve, reject) => {
+    GameLogs.aggregate([
+      {$project: {
+        player: [
+          {id: '$playerOne', threw: '$playerOneThrew'},
+          {id: '$playerTwo', threw: '$playerTwoThrew'}
+        ],
+      }},
+      {$unwind: '$player'},
+      {$match : {
+        "player.id" : new ObjectID(id)
+      }},
+      {
+        $group: {
+          _id: '$player.id',
+          player: {
+            $first: "$player.id" },
```

Comment:
are we missing a `{` somewhere here or is the indentation kind of weird?

Reply:
indentation is wierd

Reply:
Fixed